docs/classic-ui/templates-global-variables.md

```diff
@@ -0,0 +1,58 @@
+---
+html_meta:
+  "description": "Convenience global Plone variables in templates"
+  "property=og:description": "Convenience global Plone variables in templates"
+  "property=og:title": "Global variables in templates"
+  "keywords": "Plone, Classic UI, global variables, templates"
+---
+
+# Global variables in templates
+
+For convenience Plone defines a couple of global variables often used in templates.
+
+- {ref}`backend-global-utils-portal-state-label`
+- {ref}`backend-global-utils-context-state-label`
+- {ref}`backend-global-utils-plone-view-label`
+- {ref}`icons <classic-ui-icons-icon-resolver-label>`
```

Comment:
Is there a reason to add text for the link here, instead of using the default heading?

Reply:
Yest, it's a defined variable, so the name has to be icons, but the Heading is Icon Resolver ;).

Reply:
OK, thanks for explaining. This is fine.